site/content/docs/5.3/customize/sass.md

```diff
@@ -223,6 +223,34 @@ $theme-colors: map-remove($theme-colors, "info", "light", "dark");
 // etc
 ```
 
+To remove breakpoints from `$navbar-breakpoints`, use `map-remove` on `$navbar-breakpoints` or `$grid-breakpoints`. Be aware that the initial breakpoint is always skipped by the `navbar-expand-` class generation loop:
```

Comment:
I think the whole paragraph should be inside the [Navbar page](https://getbootstrap.com/docs/5.3/components/navbar/#responsive-behaviors). If so, we could avoid to mention the `$grid-breakpoints`.